cmd/lakectl/cmd/commit.go

```diff
@@ -9,31 +9,43 @@ import (
 	"github.com/treeverse/lakefs/pkg/uri"
 )
 
-var commitCreateTemplate = `Commit for branch "{{.Branch.Ref}}" completed.
+var (
+	errInvalidKeyValueFormat    = fmt.Errorf("invalid key/value pair - should be separated by \"=\"")
+	errEmptyMessage             = fmt.Errorf("commit with an empty message without specifying the \"--allow-empty-message\" flag")
```

Comment:
Don't think we need an `error` in this case.
This is an error message to the user that we can just DieFmt.
As I see it - there is a difference between an error generated by a function - like "value is require", "empty commit message is not allowed" vs the error message to the user like you specified that explains the user how to use it with the right flag.